ops/README.md

```diff
@@ -25,12 +25,19 @@ To test the changes without impacting the production server, a Vagrantfile is pr
 - Install [VirtualBox](https://www.vagrantup.com/docs/installation/)
 - Install [Vagrant](https://www.vagrantup.com/docs/installation/)
 
+#### On a Mac with an Apple Silicon processor
+
+VirtualBox is not compatible with Apple Silicon (M1…) processors. You will thus need to use the Docker provider.
+
+To that end, install Docker Desktop through a [manual install](https://hub.docker.com/editions/community/docker-ce-desktop-mac) or with `brew install homebrew/cask/docker`.
+
 ## Usage
 
-To avoid making changes on the production server by mistake, by default all commands will only affect the Vagrant development virtual machine (VM). Note that the VM needs to be started before with `vagrant up`.\
+To avoid making changes on the production server by mistake, by default all commands will only affect the Vagrant development virtual machine (VM). Note that the VM needs to be started before with `vagrant up`.  If you’re on an Apple Silicon machine or want to use Docker instead of VirtualBox, type `vagrant up --provider=docker`.
+
 To execute commands on the production server you should specify it by adding the option `--inventory ops/inventories/production.yml` to the following commands:
```

Comment:
I would rearrange the texts like this as I prefere having the development commands and THEN only, how to use it in production

```
## Usage

To avoid making changes on the production server by mistake, by default all commands will only affect the Vagrant development virtual machine (VM). Note that the VM needs to be started before with `vagrant up`.  If you’re on an Apple Silicon machine or want to use Docker instead of VirtualBox, type `vagrant up --provider=docker`.

- To set up a full [(phoenix)](https://martinfowler.com/bliki/PhoenixServer.html) server, using `inventories/dev.yml`:

```
ansible-playbook ops/site.yml
```

- To setup infrastructure only, using `inventories/dev.yml`:

```
ansible-playbook ops/infra.yml
```

- To execute commands on the production server you should specify it by adding the option `--inventory ops/inventories/production.yml` to the above commands


Setting up the production infrastructure for publishing on the shared versions repository entails decrypting a private key managed with [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html). It is decrypted with a password that we keep safe. You do not need to decrypt this specific private key on your own production server.
```